pkg/cmd/admin/image/verify-signature.go

```diff
@@ -207,12 +201,12 @@ func (o *VerifyImageSignatureOptions) Run() error {
 		}
 
 		// Verify the signed message content matches with the provided image id
-		digest, identity, signatureContentErr := o.verifySignatureContent(content)
+		identity, signatureContentErr := o.verifySignatureContent(content)
 		if signatureContentErr != nil {
 			fmt.Fprintf(o.ErrOut, "%s signature content cannot be verified: %v\n", o.InputImage, signatureContentErr)
 		}
 
-		identityError := o.verifyImageIdentity(identity, digest)
+		identityError := o.verifyImageIdentity(identity)
```

Comment:
It's perfectly fine to have multiple signatures included within an image with different identity references. The way this code is structured, prevents me from having two signatures with good verification status set at the same time, if they have different identity references.